src/ray/gcs/redis_context.cc

```diff
@@ -11,6 +11,22 @@ extern "C" {
 // TODO(pcm): Integrate into the C++ tree.
 #include "state/ray_config.h"
 
+namespace {
+
+/// A helper function to call the callback and delete it from the callback
+/// manager if necessary.
+void ProcessCallback(int64_t callback_index, const std::vector<std::string> &data) {
+  if (callback_index >= 0) {
+    bool delete_callback =
+        ray::gcs::RedisCallbackManager::instance().get(callback_index)(data);
+    // Delete the callback if necessary.
+    if (delete_callback) {
+      ray::gcs::RedisCallbackManager::instance().remove(callback_index);
+    }
+  }
+}
+}
```

Comment:
nit: leave a blank line to be symmetric

Reply:
Hmm, lint actually complains about this.